### PR TITLE
[help] fix inconsistencies

### PR DESCRIPTION
**Pull request by lalo**

### CODE REVIEW

File: test/train-sets/ref/help.stdout [hunk 1]
```diff
@@ -269,86 +265,6 @@ Contextual Bandit Exploration with Action Dependent Features:
   --bag arg             bagging-based exploration
   --greedify            always update first policy once in bagging
   --first_only          Only explore the first action in a tie-breaking event
-Contextual Bandit Exploration with Action Dependent Features:
```
Comment (lalo) [inline test/train-sets/ref/help.stdout:272]: everything deleted in this file has to be fixed

File: test/train-sets/ref/help.stdout [hunk 2]
```diff
@@ -225,10 +225,6 @@ Bootstrap:
 Continuous action tree with smoothing:
   --cats arg            Continuous action tree with smoothing
   --cats_pdf arg        Continuous action tree with smoothing (pdf)
-Continuous action tree with smoothing:
```
Comment (lalo) [inline test/train-sets/ref/help.stdout:228]: why is this duplicated?

File: test/train-sets/ref/help.stdout [hunk 3]
```diff
@@ -443,8 +356,6 @@ Gradient Descent options:
   --sparse_l2 arg (=0, ) use per feature normalized updates
   --l1_state arg (=0, )  use per feature normalized updates
   --l2_state arg (=1, )  use per feature normalized updates
-Continuous actions:
```
Comment (lalo) [inline test/train-sets/ref/help.stdout:446]: why is this duplicated?

File: test/train-sets/ref/help.stdout [hunk 4]
```diff
@@ -269,86 +265,6 @@ Contextual Bandit Exploration with Action Dependent Features:
...
-                                {ips,dr,mtr}. Default: mtr
-Contextual Bandit Exploration with Action Dependent Features:
-  --cb_explore_adf              Online explore-exploit for a contextual bandit 
-                                problem with multiline action dependent 
-                                features
-  --epsilon arg                 epsilon-greedy exploration
-  --synthcover                  use synthetic cover exploration
-  --synthcoverpsi arg (=0.1, )  exploration reward bonus
-  --synthcoversize arg (=100, ) number of policies in cover
-Continuous actions:
```
Comment (lalo) [inline test/train-sets/ref/help.stdout:345]: why is this duplicated?

**Review by jackgerrits — APPROVED** [review]

File: test/RunTests [hunk 5]
```diff
@@ -2109,7 +2109,7 @@ python3 ./cluster_test.py --vw ../build/vowpalwabbit/vw --spanning_tree ../build
     train-sets/ref/cbzo_linear_invert_hash.txt
 
 # Test 271: vw --help with filtering
-{VW} --cb_adf --rank_all -d train-sets/cb_test.ldf -p cb_adf_dr.predict --cb_type dr --help
+{VW} --cb_adf --help
```
Comment (olgavrou) [inline test/RunTests:2112]: test no: 269 after merging #2741

**Review by olgavrou — APPROVED** [review]

**Pull request merged**